hedera-mirror-rest/model/transaction.js

```diff
@@ -43,6 +45,7 @@ class Transaction {
     this.entityId = transaction.entity_id;
     this.initialBalance = transaction.initial_balance;
     this.maxFee = transaction.max_fee;
+    this.maxCustomFees = new CustomFeeLimits(transaction.max_custom_fees ?? []);
```

Comment:
since `null` check is done in `CustomFeeLimits` constructor
```suggestion
    this.maxCustomFees = new CustomFeeLimits(transaction.max_custom_fees);
```